src/LibVLCSharp.WPF/ForegroundWindow.cs

```diff
@@ -85,7 +85,7 @@ void Background_Loaded(object sender, RoutedEventArgs e)
 
             _wndhost.Closing += Wndhost_Closing;
             _bckgnd.SizeChanged += Wndhost_SizeChanged;
-            _wndhost.LocationChanged += Wndhost_LocationChanged;
```

Comment:
Is `Wndhost_LocationChanged` still used then ? Can it be removed ?

Reply:
I thought Wndhost_LocationChanged is no longer be needed.I'm agree with to remove it. @mfkl